index.html

```diff
@@ -9142,6 +9161,252 @@
+    to QUIC stream.</p>
+    <section id="rtcquicstream-overview*">
+      <h3>Overview</h3>
+      <p>An <code><a>RTCQuicStream</a></code> instance is associated to
+      an <code><a>RTCQuicTransport</a></code> instance.</p>
+    </section>
+    <section id="rtcquicstream-operation*">
+      <h3>Operation</h3>
+      An <code><a>RTCQuicStream</a></code> instance is created by the
+      <code><a>RTCQuicTransport</a></code> with a QUIC stream underneath.
+      When writing, the <code><a>RTCQuicStream</a></code>  will buffer
+      the data if the transport underneath is not writable. When receiving,
+      the received data will be buffered and methods <code>read()</code>
+      and <code>readInto()</code> read the data from the buffer.</p>
+    </section>
+    <section id="rtcquicstream-interface-definition*">
+      <h3>Interface Definition</h3>
+      <div>
+        <pre class="idl">
+interface RTCQuicStream {
+    void                  write(Uint8Array data);
+    void                  finish(optional Uint8Array data);
+    Promise&lt;void&gt;   waitForWritable(unsigned long amount,
+                                          optional unsigned long maxBufferedAmount);
+    Uint8Array            read();
+    void                  readInto(Uint8Array data);
+    Promise&lt;void&gt;   waitForReadableOrFinished(unsigned long amount,
+                                                    optional unsigned long maxBufferedAmount);
+    void                  close();
+                        attribute EventHandler             onacked;
+                        attribute EventHandler             onclosed;
+};</pre>
+      </div>
+    </section>
+    <section>
+      <h2>Attributes</h2>
+      <dl data-link-for="RTCQuicStream" data-dfn-for="RTCQuicStream" class=
+      "attributes">
+        <dt><dfn><code>onacked</code></dfn> of type <span class=
+        "idlAttrType"><a>EventHandler</a></span></dt>
+        <dd>
+          <p>This event handler, of event handler event type
+          <code><a>acked</a></code>, <em class="rfc2119" title="MUST">MUST</em>
+          be fired on reception of an ACK from the remote side.</p>
+        </dd>
+        <dt><dfn><code>onclosed</code></dfn> of type <span class=
+        "idlAttrType"><a>EventHandler</a></span></dt>
+        <dd>
+          <p>This event handler, of event handler event type <code>closed</code>,
+          <em class="rfc2119" title="MUST">MUST</em> be fired when the stream is
+          closed by either side.</p>
+        </dd>
+      </dl>
+    </section>
+    <section>
+      <h2>Methods</h2>
+      <dl data-link-for="RTCQuicStream" data-dfn-for="RTCQuicStream" class=
+      "methods">
+        <dt><code>write</code></dt>
+        <dd>
+          <p><dfn>write()</dfn> writes the data to local buffer and send the data when the
+          transport underneath is writeable. If the buffer is full, it will fail.</p>
+          <table class = "parameters">
```

Comment:
Should say that one must call waitForWritable before calling write.   And we should be careful about using the word "writable" in the method waitForWritable, it means "there is room in the write buffer".